### PR TITLE
Revert "LPS-153384 [Raylife D2C] Incorrect Product Listings"

### PR DESCRIPTION
**Pull request by eliasantosf**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js:1]: Please, remove the no-console line.

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js [hunk 2]
```diff
@@ -15,20 +16,20 @@
...
+	const catalog = await axios.get(
+		`${headlessAPI}/catalogs?filter=name%20eq%20%27Raylife%27`
 	);
 
-	const channelId = channel?.data?.items[0]?.id;
+	const catalogId = catalog?.data?.items[0]?.id;
 
 	const {data} = await axios.get(
-		`${headlessAPI}/${channelId}/products?nestedFields=skus,catalog&page=1&pageSize=50`
+		`${headlessAPI}/products?filter=catalogId%20eq%20${catalogId}&nestedFields=skus,catalog&page=1&pageSize=50`
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js:32]: Please, adjuts this line to: 
```suggestion
		`${headlessAPI}/products?filter=catalogId eq ${catalogId}&nestedFields=skus,catalog&page=1&pageSize=50`
```

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js [hunk 3]
```diff
@@ -15,20 +16,20 @@
...
+const headlessAPI = 'o/headless-commerce-admin-catalog/v1.0';
 
 /**
  * @returns {Promise<ProductQuote[]>)} Array of Product Quote
  */
 export async function getProductQuotes() {
-	const channel = await axios.get(
-		`http://localhost:8080/o/headless-commerce-admin-channel/v1.0/channels?search=raylife`
+	const catalog = await axios.get(
+		`${headlessAPI}/catalogs?filter=name%20eq%20%27Raylife%27`
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js:26]: Please, adjust this line to:
```suggestion
		`${headlessAPI}/catalogs?filter=name eq 'Raylife'`
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
Hey @eliasantosf , I add some comments.

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js (hunk 1 above)
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js (hunk 2 above)
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js (hunk 3 above)
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/extra/remote-app/src/routes/get-a-quote/services/CommerceCatalog.js]: Done!

**Review by silasbsi — APPROVED** [review]
LGTM :fire:

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**